dbt/README.md

```diff
@@ -284,7 +306,7 @@ dbt docs serve
 
 Then, navigate to http://localhost:8080 to view the site.
 
-<h3 id="how-to-add-a-new-model"> ➕ How to add a new model </h3>
+<h2 id="how-to-add-a-new-model"> ➕ How to add a new model </h2>
```

Comment:
Also reworked some of the heading levels just to make the outline view a bit clearer.